lib/ansible/modules/system/filesystem.py

```diff
@@ -16,21 +16,22 @@
 author:
 - Alexander Bulimov (@abulimov)
 module: filesystem
-short_description: Makes file system on block device
+short_description: Makes filesystem
 description:
-  - This module creates file system.
+  - This module creates filesystem.
 version_added: "1.2"
 options:
   fstype:
     choices: [ btrfs, ext2, ext3, ext4, ext4dev, lvm, reiserfs, xfs ]
     description:
-    - File System type to be created.
+    - FileSystem type to be created.
     - reiserfs support was added in 2.2.
     - lvm support was added in 2.5.
+    - since 2.5, I(dev) can be a regular file.
     required: yes
   dev:
     description:
-    - Target block device.
+    - Target block device or regular file.
```

Comment:
since its  not really a 'regular file' I would change it to 'Target path to device or image file' or similar

Reply:
Done